cli/formatter/go-pretty/extra/002_add_interface_for_001.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/table/writer.go b/table/writer.go
```

Comment:
Let's merge the patch with `001.patch`. It looks like a single feature.

Reply:
Merged